tests/Unit/Evolution/Systems/NewtonianEuler/Test_Characteristics.cpp

```diff
@@ -0,0 +1,85 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "Evolution/Systems/NewtonianEuler/Characteristics.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "tests/Unit/Pypp/CheckWithRandomValues.hpp"
+#include "tests/Unit/Pypp/SetupLocalPythonEnvironment.hpp"
+
+namespace {
+
+enum Axes {
+  x_axis = 1,
+  y_axis,
+  z_axis,
+};
+
+template <size_t Dim>
+tnsr::i<DataVector, Dim> normal_along_axis(
```

Comment:
Factor this into a file, possible in DomainHelpers.hpp and do every direction using the `Direction<Dim>` type.